tools/cli/internal/cli/split/split.go

```diff
@@ -59,7 +59,7 @@ func (o *Opts) Run() error {
 
 	// make a copy of the oas to avoid modifying the original document when applying filters
 	for _, version := range versions {
-		oasCopy, err := duplicateOas(oas)
+		oasCopy, err := duplicateOas(oas) // @Todo this can go outside the loop maybe !?
```

Comment:
yeah, I think we need to duplicate one per file we want to generate and operate the filters on it

Reply:
The good news is that I compared the v2024-05-30.json created with foascli with the one in production and the descriptions fields of the responses matched

Reply:
wohooooo 🚀🚀🚀amazing news

Reply:
we can add this comparison as e2e, get a file we generated in the past and compare. I'll write a ticket for the e2e.